main.js

```diff
@@ -122,6 +130,16 @@ function validateSeconds() {
   }
 }
 
+function formatNumber(number) {
+  var parsedNumber = parseInt(number);
+  if (parsedNumber < 10) {
+    var stringNumber = `0` + parsedNumber;
+    return stringNumber;
+  } else {
+    return number;
```

Comment:
good catch on the undefined issues for numbers greater than 10! need this return statement.